raiden/raiden_event_handler.py

```diff
@@ -455,14 +455,22 @@ def handle_contract_send_channelupdate(
             )
             our_signature = raiden.signer.sign(data=non_closing_data)
 
-            channel.update_transfer(
-                nonce=balance_proof.nonce,
-                balance_hash=balance_proof.balance_hash,
-                additional_hash=balance_proof.message_hash,
-                partner_signature=balance_proof.signature,
-                signature=our_signature,
-                block_identifier=channel_update_event.triggered_by_block_hash,
-            )
+            try:
+                channel.update_transfer(
+                    nonce=balance_proof.nonce,
+                    balance_hash=balance_proof.balance_hash,
+                    additional_hash=balance_proof.message_hash,
+                    partner_signature=balance_proof.signature,
+                    signature=our_signature,
+                    block_identifier=channel_update_event.triggered_by_block_hash,
+                )
+            except InsufficientEth:
```

Comment:
The event handler code submits alot of other transaction. If we're following this approach then all places where we submit transactions should handle `InsufficientEth`

Reply:
That's why it was still marked WIP, the handling of the exception in the other contract event handlers is missing.